src/server.jsx

```diff
@@ -178,6 +180,7 @@ server
           } else {
             res.status(200).send(
               `<!doctype html>
+                <script>window.env = ${serialize(runtimeConfig)};</script>
```

Comment:
I would set this up in Html.jsx

Reply:
@zotya how about this? The reason is to have all the JS that we load in one place. Please can you move it to `HTML.jsx`?

Reply:
@sneridagh I agree, i moved the code to Html.jsx